src/client/updaters/tag.js

```diff
@@ -0,0 +1,84 @@
+// borrow the slice method
+const toArray = Function.prototype.call.bind(Array.prototype.slice)
```

Comment:
Would `Array.from` fit here?

Reply:
This code runs in the browser and Array.from needs to be polyfilled for older ones, therefor I decided not to touch it. I looked in general at this stuff (eg using babel instead of buble with rollup increases package size already by ~2kb) but not at this detail level.

Reply:
Polyfill and browser support is the job of bundler/transpiler, not us! Writing legacy code also adds in the modern bundle too.

Reply:
I agree, but in the vue-app-head rfc Atinux had explicit concerns about vue-meta's package size and the polyfill for Array.from is much larger then this slice. And we dont really need all the type checking that Array.from does here, but will change it anyway :)

Reply:
Hmm, I checked but buble doesnt add a polyfill for Array.from. I will leave that to you all to decide whether thats ok or not.

Related issue: https://gitlab.com/Rich-Harris/buble/issues/81